public/e2e-test/pages/dashboards/dashboardPage.ts

```diff
@@ -1,11 +1,48 @@
-import { ClickablePageObject, ClickablePageObjectType, Selector, TestPage } from '@grafana/toolkit/src/e2e';
+import { ArrayPageObjectType, ClickablePageObjectType, PageObjectType, TestPage } from '@grafana/toolkit/src/e2e';
 
 export interface DashboardPage {
   settings: ClickablePageObjectType;
+  submenuItemLabel: ArrayPageObjectType;
+  submenuItemValueDropDownValueLink: ArrayPageObjectType;
+  submenuItemValueDropDownDropDown: PageObjectType;
+  submenuItemValueDropDownSelectedLink: PageObjectType;
+  submenuItemValueDropDownOptionText: ArrayPageObjectType;
 }
 
 export const dashboardPage = new TestPage<DashboardPage>({
   pageObjects: {
-    settings: new ClickablePageObject(Selector.fromAriaLabel('Dashboard settings navbar button')),
+    settings: 'Dashboard settings navbar button',
+    submenuItemLabel: 'Dashboard template variables submenu LabelName label',
+    submenuItemValueDropDownValueLink: 'Dashboard template variables Variable Value DropDown value link',
+    submenuItemValueDropDownDropDown: 'Dashboard template variables Variable Value DropDown DropDown',
+    submenuItemValueDropDownSelectedLink: 'Dashboard template variables Variable Value DropDown Selected link',
+    submenuItemValueDropDownOptionText: 'Dashboard template variables Variable Value DropDown option text',
   },
 });
+
+export interface AssertVariableLabelsAndComponentsArguments {
+  label: string;
+  options: string[];
+}
+
+export const assertVariableLabelsAndComponents = async (
+  page: TestPage<DashboardPage>,
+  args: AssertVariableLabelsAndComponentsArguments[]
+) => {
+  console.log('Asserting variable components and labels');
```

Comment:
```suggestion
```
?:)

Reply:
It's a way to see in the output of the tests in what step of the scenario we're. It's kind of hard to debug otherwise, if you don't start the Browser that is.